asset_dashboard/models.py

```diff
@@ -248,3 +248,192 @@ class DummyProject(models.Model):
     def __str__(self):
         """"String for representing the model object"""
         return self.name
+
+
+class GISModel(models.Model):
+    """
+    Migrations will not be created for models inheriting from this base class
+    because of allow_migrate() method in the GISRouter.
+    """
+
+    class Meta:
+        abstract = True
+        managed = False
+        app_label = 'asset_dashboard_gis'
+
+
+class NaturePreserves(GISModel):
+
+    class Meta(GISModel.Meta):
+        db_table = '"quercus"."nature_preserves"'
+
+    id = models.AutoField(primary_key=True, db_column='nature_preserves_id')
+    site_name = models.CharField(max_length=254)
+
+    def __str__(self):
+        return self.site_name
+
+
+class Buildings(GISModel):
```

Comment:
@smcalilly roughed in models for the remaining tables. The `db_table` attribute needs to be updated to quote the schema and table names, and the columns we need to use need to be defined as attributes.